### PR TITLE
[1.x] Ignore SOLV writing exceptions

### PR DESCRIPTION
**Pull request by jaimergp**
Closes https://github.com/mamba-org/mamba/issues/3312

### CODE REVIEW

**Comment by jaimergp** [comment]
FWIW, this is implemented in libmamba 2.x as a warning:

https://github.com/mamba-org/mamba/blob/a10a532fd810453d337547f35b43abe78aee5053/libmamba/src/core/package_database_loader.cpp#L117-L127

File: libmamba/src/core/repo.cpp [hunk 1]
```diff
@@ -329,7 +330,14 @@ namespace mamba
...
         if (name() != "installed")
         {
-            write_solv(solv_file);
+            try
+            {
+                write_solv(solv_file);
+            }
+            catch (const std::exception& e)
+            {
+                LOG_INFO << "Could not write libsolv solv file for " << json_file << ": " << e.what();
```
Comment (Hind-M) [inline libmamba/src/core/repo.cpp:339]: I think we can emit a warning instead (and to be consistent with mamba 2):
```suggestion
                LOG_WARNING << "Could not write libsolv solv file for " << json_file << ": " << e.what();
```

**Comment by Hind-M** [comment]
I think you can rebase and that will fix the CI :)

**Comment by jaimergp** [comment]
Let's see what happens!

**Comment by mcg1969** [comment]
@jaimergp I'd love to see this fixed, so thank you for working on it. May we consider pushing it forward?

**Comment by mcg1969** [comment]
Use cases: I have a read-only package cache with pre-computed `.solv` files

**Review by Hind-M — APPROVED** [review]
Thanks!

**Pull request merged**